modules/govuk_ci/manifests/agent/mysql.pp

```diff
@@ -0,0 +1,100 @@
+# == Class: govuk_ci::agent::mysql
+#
+# Installs and configures mysql-server
+#
+class govuk_ci::agent::mysql {
+  contain ::govuk_mysql
```

Comment:
This is a bare class. I think it should be `::govuk_mysql::server` instead. Then we wouldn't also need to include `::mysql::server`, because that's included in the aforementioned class.